src/etc/natvis/libcore.natvis

```diff
@@ -6,34 +6,29 @@
       <Item Name="[ptr]">pointer</Item>
     </Expand>
   </Type>
+
   <Type Name="core::ptr::Shared&lt;*&gt;">
     <DisplayString>{{ Shared {pointer} }}</DisplayString>
     <Expand>
       <Item Name="[ptr]">pointer</Item>
     </Expand>
   </Type>
+
   <Type Name="core::option::Option&lt;*&gt;">
-    <DisplayString Condition="RUST$ENUM$DISR == 0x0">{{ None }}</DisplayString>
-    <DisplayString Condition="RUST$ENUM$DISR == 0x1">{{ Some {__0} }}</DisplayString>
+    <DisplayString Condition="RUST$ENUM$DISR == 0x0">None</DisplayString>
+    <DisplayString Condition="RUST$ENUM$DISR == 0x1">Some({__0})</DisplayString>
     <Expand>
-      <Item Name="[size]" ExcludeView="simple">(ULONG)(RUST$ENUM$DISR != 0)</Item>
+      <Item Name="[size]" ExcludeView="simple">(unsigned long)(RUST$ENUM$DISR != 0)</Item>
```

Comment:
```suggestion
      <Item Name="[len]" ExcludeView="simple">(unsigned long)(RUST$ENUM$DISR != 0)</Item>
```
If size is changed to len in other cases, then it would make sense to change it here as well.

Reply:
Thanks. Fixed.